src/components/tab-nav/tab-nav.tsx

```diff
@@ -234,9 +242,17 @@ export class TabNav {
     event.preventDefault();
   }
 
-  @Listen("calciteTabsActivate") activateTabHandler(event: CustomEvent<void>): void {
+  @Listen("calciteTabsActivate")
+  activateTabHandler(event: CustomEvent<void>): void {
     this.calciteTabChange.emit();
+    event.stopPropagation();
+    event.preventDefault();
```

Comment:
Is there a reason preventDefault is called here? I don't think there's a default action for tabsactivate right?